README.md

```diff
@@ -17,24 +19,22 @@ The current implementation supports the following for CStor Volumes:
 
 ### Prerequisites
 
-Before setting up OpenEBS CStor CSI driver make sure your Kubernetes Cluster
+Before setting up OpenEBS cStor CSI driver make sure your Kubernetes Cluster
```

Comment:
@prateekpandey14 to maintain a single source for setup instructions. Can you remove the following sections: 

- Prerequisites
- Setup OpenEBS CStor CSI Driver
- Provision a cStor volume using OpenEBS cStor CSI driver
- Snapshot And Clone cStor Volume using OpenEBS cStor CSI Driver

You can instead mention that. 

```
Refer to https://github.com/openebs/cstor-operators for detailed usage instructions. 
```

Let this page/repo only focus on how does it work. 

Change the sections on `Snapshot And Clone cStor` to also document the steps in involved in how does it work.

Reply:
Cleaned the mentioned section , will create an issue to add the section on `Snapshot and Clone cStor` workflow doc.